shared/oae/api/oae.api.js

```diff
@@ -150,17 +150,34 @@ define(['oae.api.admin', 'oae.api.authentication', 'oae.api.config', 'oae.api.co
         };
 
 
-        //////////////////////////
-        // Terms and Conditions //
-        //////////////////////////
+        /////////////////////////////////////////
+        // User details & Terms and Conditions //
+        /////////////////////////////////////////
 
         /**
-         * Trigger the Terms and Conditions widget if the Terms and Conditions
-         * need to be accepted before using the system.
+         * Trigger the user details widgets if the user needs to provide additional profile information
```

Comment:
This also suggest that the user details widget contains will take care of both providing additional profile info and accepting Terms and Conditions